projects/packages/my-jetpack/_inc/components/connected-product-card/index.tsx

```diff
@@ -74,7 +74,7 @@ const ConnectedProductCard: FC< ConnectedProductCardProps > = ( {
 			return;
 		}
 
-		activate();
+		activate( {} );
```

Comment:
I'm creating a separate maintenance task to fix that 😓

Reply:
Thank you!